### PR TITLE
Feature/add id attribute

### PR DESCRIPTION
**Pull request by omarzoghayyer**
### What does this pull request aim to achieve? 💯

Added id attribute to both sign in and sign up pages.

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by omarzoghayyer** [comment]
@silenceway @idjevm could you see if thats the right way of adding these attributes?

**Comment by doheekim93** [comment]
@omarzoghayyer Hey, I checked the id's and I think you used the SU_ prefix id's for the Sign In elements. Those should have the SI_ prefix id's. Could you double check those please? Thanks!

**Comment by omarzoghayyer** [comment]
@doheekim93 Yes thats right I accidentally assigned sign in with SU. It should be changed now.

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -508,14 +508,14 @@ const Login = ({ isLoginForm, forgotPassword }) => {
                       </AuthLink>
                     </p>
                     <p>
-                      <AuthLink to="/auth/signup">
+                      <AuthLink  id= "SU_JN" to="/auth/signup">
```
Comment (nr431207) [inline client/src/pages/Login.js:511]: unnecessary space between `= "`

**Comment by doheekim93** [comment]
Hey @omarzoghayyer, you figured out the social media stuff! Everything looks good from my end!

**Comment by doheekim93** [comment]
@idjevm Oh good catch!! Thank youu

**Pull request merged**